### PR TITLE
feat: added notification for request execution statuses

### PR DESCRIPTION
**Pull request by HuziMuzi**
image

### CODE REVIEW

File: src/bll/appReducer.ts [hunk 1]
```diff
@@ -1,15 +1,40 @@
-import {createSlice} from '@reduxjs/toolkit';
+import {createSlice, PayloadAction} from '@reduxjs/toolkit';
+import {AppMessageType} from '@customTypes/Notification-types';
```
Comment (antoniosk10) [inline src/bll/appReducer.ts:2]: Notification-types -> notificationTypes

File: src/components/Notification/index.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    stopTimer();
+    const id: ReturnType<typeof setInterval> = setInterval(() => {
+      dispatch(setAppLastMessage());
+    }, 2900);
+    setTimerId(id);
+  };
+
+  useEffect(() => {
+    messages.length ? startTimer() : stopTimer();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (antoniosk10) [inline src/components/Notification/index.tsx:37]: why does it need here?

File: src/bll/appReducer.ts [hunk 3]
```diff
@@ -1,15 +1,40 @@
...
+      action: PayloadAction<{text: string; severity: 'error' | 'success'}>,
+    ) {
+      const newMessage: AppMessageType = {
+        id: Math.random(),
+        severity: action.payload.severity,
+        text: action.payload.text,
+      };
+      state.messages.push(newMessage);
+    },
+    setAppLastMessage(state) {
```
Comment (antoniosk10) [inline src/bll/appReducer.ts:29]: maybe removeLastMessage?

File: src/components/Notification/index.tsx (hunk 2 above)
Comment (HuziMuzi) [reply src/components/Notification/index.tsx:37]: ESlint wants me to add 'startTimer' and 'stopTimer' as dependencies. These dependencies are not needed there. When I add them, the code stops working.

**Review by KhaninPasha0488 — APPROVED** [review]

**Review by antoniosk10 — APPROVED** [review]

**Review by KhaninPasha0488 — APPROVED** [review]

**Pull request merged**